.github/workflows/generate-next-preview.yml

```diff
@@ -0,0 +1,609 @@
+            echo "❌ RELEASE BLOCKED: Sequential integration conflicts detected in branches:$CONFLICTED_BRANCHES"
+            echo "❌ Conflicts occur when applying breaking changes in numerical order"
+            echo "❌ No preview release will be generated until all conflicts are resolved"
+
+            echo "❌ **Release blocked due to conflicts!**" >> $GITHUB_STEP_SUMMARY
+            echo "" >> $GITHUB_STEP_SUMMARY
+            echo "Conflicted branches:$CONFLICTED_BRANCHES" >> $GITHUB_STEP_SUMMARY
+          fi
+
+      - name: Create ephemeral integration branch
+        if: steps.discovery.outputs.found == 'true' && steps.validation.outputs.all_clean == 'true'
+        run: |
+          # Create temporary integration branch from latest develop
+          INTEGRATION_BRANCH="ephemeral-integration-$(date +%Y%m%d%H%M%S)"
+          echo "INTEGRATION_BRANCH=$INTEGRATION_BRANCH" >> "$GITHUB_ENV"
+
+          git checkout -b "$INTEGRATION_BRANCH" origin/develop
+          echo "📝 Created ephemeral integration branch: $INTEGRATION_BRANCH"
+
+      - name: Cherry-pick breaking changes
+        if: steps.discovery.outputs.found == 'true' && steps.validation.outputs.all_clean == 'true'
+        id: integration
+        run: |
+          echo "🚀 Cherry-picking breaking changes (all branches pre-validated as conflict-free)..."
+
+          BREAKING_CHANGE_METADATA=""
+          INTEGRATED_COMMITS=""
+          TOTAL_COMMITS=0
+
+          for branch in $BC_BRANCHES; do
+            echo "📦 Processing branch: $branch"
+
+            # Get commits from this branch not in develop
+            COMMITS=$(git rev-list --reverse "origin/develop..origin/$branch" 2>/dev/null || echo "")
+
+            if [[ -z "$COMMITS" ]]; then
+              echo "⚠️ No new commits found in $branch"
+              continue
+            fi
+
+            # Cherry-pick each commit in chronological order
+            for commit in $COMMITS; do
+              COMMIT_MSG=$(git log -1 --format="%s" "$commit")
+              COMMIT_BODY=$(git log -1 --format="%b" "$commit")
+              COMMIT_AUTHOR=$(git log -1 --format="%an <%ae>" "$commit")
+              echo "Applying commit: $commit ($COMMIT_MSG)"
+
+              # Cherry-pick will succeed since we pre-validated
+              git cherry-pick "$commit"
+              INTEGRATED_COMMITS="$INTEGRATED_COMMITS $commit"
+              TOTAL_COMMITS=$((TOTAL_COMMITS + 1))
+              echo "✅ Successfully applied: $commit"
+
+              # Parse breaking change metadata if this is a breaking change commit
+              if [[ "$COMMIT_MSG" == *"!"* ]] && [[ "$COMMIT_BODY" == *"BREAKING CHANGE:"* ]]; then
+                # Extract structured metadata
+                CATEGORY=$(echo "$COMMIT_BODY" | grep "Breaking-Change-Category:" | cut -d: -f2- | xargs || echo "")
+                IMPACT=$(echo "$COMMIT_BODY" | grep "Breaking-Change-Impact:" | cut -d: -f2- | xargs || echo "")
```

Comment:
I left a comment on the documentation section, but I'd prefer not having these sections.